ens/constants.py

```diff
@@ -1,10 +1,20 @@
+from typing import (
+    cast,
+)
+
+from eth_typing import (
+    Address,
+)
+from hexbytes import (
+    HexBytes,
+)
 
 ACCEPTABLE_STALE_HOURS = 48
 
 AUCTION_START_GAS_CONSTANT = 25000
 AUCTION_START_GAS_MARGINAL = 39000
 
-EMPTY_SHA3_BYTES = b'\0' * 32
-EMPTY_ADDR_HEX = '0x' + '00' * 20
+EMPTY_SHA3_BYTES = cast(HexBytes, b'\0' * 32)
+EMPTY_ADDR_HEX = cast(Address, '0x' + '00' * 20)
```

Comment:
I don't think this is accurate...

https://github.com/ethereum/eth-typing/blob/e415a1c70d0751283f048d6365165839cdd18540/eth_typing/evm.py

Suggests it should be `HexAddress` or `AnyAddress`